Server/IT-Community.Server.Infrastructure/Services/TagsService.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AutoMapper;
+using IT_Community.Server.Core.DataAccess;
+using IT_Community.Server.Core.Entities;
+using IT_Community.Server.Infrastructure.Dtos.TagsDTOs;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace IT_Community.Server.Infrastructure.Services
+{
+    public class TagsService : ITagsService
+    {
+        private readonly IUnitOfWork _unitOfWork;
+        private readonly IMapper _mapper;
+
+        public TagsService(IUnitOfWork unitOfWork, IMapper mapper)
+        {
+            _unitOfWork = unitOfWork;
+            _mapper = mapper;
+        }
+
+        public List<TagDto> GetTags()
+        {
+            var tags = _unitOfWork.TagRepository.GetAll();
+            var tags1 = tags.Select(t => _mapper.Map(t, new TagDto())).ToList();
```

Comment:
better to use collection mapping
example: var collection = _mapper.Map<List<CollectionItem>>(collectionToMap);